source/test/TestApp/PlayFabEventTest.h

```diff
@@ -57,11 +64,19 @@ namespace PlayFabUnit
             static int eventFailCount;
             static std::string eventFailLog;
 
+            static const unsigned int numThreads = 6;
```

Comment:
>unsigned int [](start = 25, length = 12)

nit: instead of using `unsigned int`, it would be shorter (i.e. easier to read) to use a sized-int type ala uint32_t